multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1505,6 +1506,9 @@ def _fit(
         # save artifacts for the current running, except for model checkpoint, which will be saved in trainer
         self.save(save_path, standalone=standalone)
 
+        # After first DDP run, change following DDP runs to single GPU
+        num_gpus, strategy = run_ddp_only_once(num_gpus, strategy)
```

Comment:
Need to move this above line 1502 because `num_gpus` needs update in `config`.

Reply:
But at next time it's loaded it will run in single GPU mode?